AuctionSite.Bl/User/Validators/EmailValidator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using AuctionSite.BL.Common;
+using AuctionSite.Shared;
+using AuctionSite.Shared.BindingModel;
+using AuctionSite.Shared.Enums;
+using AuctionSite.Shared.Exceptions;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Net.Mail;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace AuctionSite.BL.User.Validators
+{
+    public class EmailFormatValidator : IValidator<RegisterUserBindingModel>
```

Comment:
Zbędny validator, to można osiągnąć poprzez atrybut [Email] czy coś takiego w BindingModel'u

Reply:
Można